### PR TITLE
Create main.yml

### PR DESCRIPTION
**Pull request by Jadevious**

### CODE REVIEW

**Comment by Jadevious** [comment]
Link to ticket: https://trello.com/c/2Y6hQ5Ul

**Review by Kirsten1822 — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      # Runs a single command using the runners shell
+      - name: building ui project
+        run: npm install
```
Comment (JoshKainos) [inline .github/workflows/main.yml]: You are missing these lines

- run: npm run build --if-present 
- run: npm test

Test can be omitted for now as there are no tests but a tech debt ticket will be required to include later

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      # Runs a single command using the runners shell
+      - name: building ui project
+        run: |
```
Comment (JoshKainos) [inline .github/workflows/main.yml]:
```suggestion
    - run: npm install
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      # Runs a single command using the runners shell
+      - name: building ui project
+        run: | 
+          npm install
```
Comment (JoshKainos) [inline .github/workflows/main.yml]:
```suggestion
        - run: npm run build --if-present
```

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          run: npm run build --if-present
```
Comment (JoshKainos) [inline .github/workflows/main.yml]: Remove this line

**Review by JoshKainos — CHANGES_REQUESTED** [review]

**Review by JoshKainos — APPROVED** [review]

**Pull request merged**